components/admin_console/billing/billing_subscriptions/limits.tsx

```diff
@@ -45,7 +45,7 @@ const Limits = (props: Props): JSX.Element | null => {
     const usage = useGetUsage();
     const openSalesLink = useOpenSalesLink();
 
-    if (!isCloudFreeEnabled || !limitsLoaded || !subscriptionProduct || subscriptionProduct.sku === CloudProducts.STARTER_LEGACY) {
+    if (!isCloudFreeEnabled || !limitsLoaded || !subscriptionProduct || subscriptionProduct.sku === CloudProducts.STARTER_LEGACY || subscription?.is_free_trial) {
```

Comment:
Should the check here be for Enterprise instead? They are on the free trial for Enteprise and if we subscribe for Enterprise it shouldn't show the limits either (missed it in the initial review)

Reply:
changed.

Reply:
Thanks!